administrator/language/de-DE/com_config.ini

```
@@ -60,19 +60,27 @@ COM_CONFIG_FIELD_DATABASE_NAME_LABEL="Datenbank"
 COM_CONFIG_FIELD_DATABASE_PASSWORD_DESC="Das Passwort für den Zugriff auf die Datenbank.<br /><br /><strong>Warnung:</strong><br />Dieses Feld sollte nicht verändert werden, außer in speziellen Fällen, wie z.&#8239;B. beim Umzug der Datenbank zu einem neuen Webhoster."
 COM_CONFIG_FIELD_DATABASE_PASSWORD_LABEL="Passwort"
 COM_CONFIG_FIELD_DATABASE_PREFIX_LABEL="Präfix"
+COM_CONFIG_FIELD_DATABASE_TYPE_DESC="Der zu nutzende Datenbanktyp, der während der Installation eingegeben wurde.<br /><br /><strong>Warnung:</strong><br />Dieses Feld sollte nicht verändert werden, außer in speziellen Fällen, wie z.&#8239;B. beim Umzug der Datenbank zu einem neuen Webhosting-Provider."
 COM_CONFIG_FIELD_DATABASE_TYPE_LABEL="Typ"
 COM_CONFIG_FIELD_DATABASE_USERNAME_LABEL="Benutzer"
 COM_CONFIG_FIELD_DEBUG_CONST="Konstante"
 COM_CONFIG_FIELD_DEBUG_CONST_LANG_DESC="Auswählen, ob die Sprachkonstante oder der Sprachwert angezeigt werden sollen, wenn die Sprache debuggt wird."
 COM_CONFIG_FIELD_DEBUG_CONST_LANG_LABEL="Sprachanzeige"
+COM_CONFIG_FIELD_DEBUG_LANG_DESC="Auswählen, ob die Fehleranzeiger (<strong>** ... **</strong>) oder (<strong>?? ... ??</strong>) für die Joomla-Sprache angezeigt werden sollen. „Sprache debuggen“ funktioniert auch ohne die Aktivierung von „System debuggen“, zeigt dann allerdings nicht die benötigten zusätzlichen Informationen an, die bei der Korrektur der Fehler hilfreich wären."
```

```suggestion
COM_CONFIG_FIELD_DEBUG_LANG_DESC="Auswählen, ob die Fehleranzeiger (<strong>** ... **</strong>) oder (<strong>?? ... ??</strong>) für die Joomla-Sprache angezeigt werden sollen. „Sprache debuggen“ funktioniert auch ohne die Aktivierung von „System debuggen“, zeigt dann allerdings nicht die benötigten zusätzlichen Informationen an, die bei der Korrektur der Fehler hilfreich wären."
```
Das ist keine Fehleranzeiger. "??" Steht dann bei jedem String. Man kann damit erkennen ob ein string via language string kommt oder nicht.

---

@zero-24 see 6c2bf51bfde6409c9ecf2b442511f87650eb445b